### ISSUE TITLE
More manipulations

### ISSUE THREAD

**Issue by ChiragJhawar** [stla/rAmCharts4#3]
Hello Stéphane, I would like to thank you for this amazing repo. I am trying to use the draggable bar chart on a website that I am developing for education in bayesian interfaces. I was wondering if there are more manipulations possible, for eg: having the user input a value in a table to change the the chart.

**Comment by stla** [comment]
Hello @ChiragJhawar . Currently this is not possible. I will try to implement this feature today. But this will be possible only with Shiny. Can you use a Shiny app on your website?

**Comment by ChiragJhawar** [comment]
Hello, thank you for responding. Yes, I can use a shiny app on my website. In fact, that is what the goal is.

**Comment by stla** [comment]
Perfect. Wait a minute, this is done and this works :)

**Comment by stla** [comment]
You can install the latest version. The example below is taken from `?updateAmBarChart`. 

![rAmCharts4_updateAmBarChart](https://user-images.githubusercontent.com/4466543/96334856-218be880-1074-11eb-8bb5-cc11d9650bdd.gif)

**Comment by ChiragJhawar** [comment]
Hello, thank you for the prompt response. I am sorry for the delayed one. I'd like to understand more about the function and the package, is there anyway we could connect? Thank You

**Comment by stla** [comment]
Hi. We can discuss here if you want, so that the folks can read the discussion.  For a question about coding, please use StackOverflow, I will reply.

**Comment by ChiragJhawar** [comment]
Sure, I am trying to nest the bar graphs instead of grouping them. In the Javascript version of AC4, It meant rendering one plot over another, is it the same case in this package?

**Comment by stla** [comment]
You mean you want a *stacked* bar chart? This is not implemented yet.

**Comment by ChiragJhawar** [comment]
Hello, that is exactly what I mean. I am wondering if I could just render two plots on each other or maybe put negative distance between 2 grouped charts?
Again, sorry if these seem dumb, I am new to R.

**Comment by stla** [comment]
No, you can't do that. I have to implement it. Perhaps I will do it soon. Cheers.
